app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -20,6 +21,7 @@ module Api::V2::TaxonomiesController
       param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
       param :media_ids, Array, N_("Media IDs"), :required => false
       param :config_template_ids, Array, N_("Provisioning template IDs"), :required => false
+      param :provisioning_template_ids, Array, N_("Provisioning template IDs"), :required => false
```

Comment:
@eLobato I've added just a FIXME comment to locate the place. We should add support for deprecated options into Apipie first. Then this is information could be reused in docs and Hammer. I filed issue as a reminder  http://projects.theforeman.org/issues/10869